template/script.sh.erb

```diff
@@ -47,6 +47,7 @@ module load project/ondemand xalt/latest <%= context.version %>
 <%- else -%>
 module load ondemand/project xalt/latest <%= context.version %>
 <%- end -%>
+module load spark/<%= context.spark_version %>
```

Comment:
Do we need to remove the other spots where `spark/<version>` is loaded? It seems to be loaded & booted in `before.sh.erb`.